006-vortexa-tech-challenge/src/materials-pie-chart.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import { ChartDatum, RampData } from "./types";
+
+import { ErrorBoundary } from "react-error-boundary";
+import React from "react";
```

Comment:
Some import sorting is needed 😊 

```suggestion
import React from "react";
import * as ReCharts from "recharts";
import { ErrorBoundary } from "react-error-boundary";

import { ChartDatum, RampData } from "./types";
```